CHANGELOG.md

```diff
@@ -98,6 +98,13 @@ There are specific things that you need to change regarding your Form or `Virtus
 - Very rarely, when defining a an attribute of type `Rails::Engine`, you need to change `attribute :attr_name, Rails::Engine` to `attribute :attr_name, Rails::Engine, {}`. This is because we want to preserve the method signature against `ActiveModel::Attributes` for the `attribute` class method intead of the legacy `Virtus.model`. There is a limitation in the Ruby language that if the method has default values for the previous arguments and defines keyword arguments, the last argument will always receive a `respond_to?(:to_hash)` call to it which doesn't work for `Rails::Engine` (you can try it out in the Rails console by calling `Rails::Engine.respond_to?(:to_hash)`).
 - Test all your form and command classes thoroughly to notice any differences between the two implementations. The new layer is a bit more "robust" with some of the type castings, so some things may break during the migration in case you have relied on some of the oversights within `Virtus`.
 
+### Added
+* [#8012](https://github.com/decidim/decidim/pull/8012) Participatory space to comments, to fix the statistics. Use
+`rake decidim_comments:update_participatory_process_in_comments` to migrate existing comments to the new structure.
+You can manually regenerate the docs by running: ```rake decidim_api:generate_docs```
+
+* [#8866](https://github.com/decidim/decidim/pull/8866) Possibility to delete every current participants of a participatory
+space when adding some via csv, with warning message and confirmation dialog box
```

Comment:
I'm not sure this is needed since no action is required.

Reply:
Yes, you're right @armandfardeau. As our [docs](https://docs.decidim.org/en/develop/guide_conventions/#_changelog) say:

> We will not ask CHANGELOG for all the changes make on this repository. We will ask for CHANGELOG instructions only for special changes that really need some actions on part of developers/implementers or something to comunicate on the releases notes

On this case, as it's a feature that adds a new feature and it's pretty visible already, I'd said there's no need to add this to the CHANGELOG